services/kvstore/src/main/java/tech/pegasys/pantheon/services/kvstore/RocksDBStats.java

```diff
@@ -0,0 +1,198 @@
+package tech.pegasys.pantheon.services.kvstore;
```

Comment:
What do you think about moving this to: `tech.pegasys.pantheon.metrics.rocksdb`.  We have a similar paradigm for vertx utilities in `tech.pegasys.pantheon.metrics.vertx`.